internal/cmd/organization/organization.go

```diff
@@ -15,9 +15,9 @@ import (
 func NewCmd(p *print.Printer) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "organization",
-		Short: "Provides functionality regarding organizations",
+		Short: "Provides functionality for organizations",
 		Long: fmt.Sprintf("%s\n%s",
-			"Provides functionality regarding organizations.",
+			"Provides functionality for organizations.",
```

Comment:
And here, `Manage members and roles of an organization` or just `Manage organizations`?

Reply:
The second one

Reply:
To align with the other descriptions, I need to use the 3rd person singular verb: Manages. 
Would "Manages organizations" work for you? It sounds a bit weird :/

Reply:
Sounds good to me